CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * Add support for embedded objects in the C API. ([#5408](https://github.com/realm/realm-core/issues/5408))
 * Added `realm_object_to_string()` support for c_api. ([#5414](https://github.com/realm/realm-core/issues/5414))
 * Added `ObjectStore::rename_property()` support for c_api. ([#5424]https://github.com/realm/realm-core/issues/5424)
+* Removed unused sync protocol error `disabled_session`. ([#5421]https://github.com/realm/realm-core/issues/5421)
```

Comment:
Suggest: Remove deprecated sync protocol errors `disabled_session` and `superseded`